compiler-core/src/parse/snapshots/gleam_core__parse__tests__attributes_with_no_definition.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: compiler-core/src/parse/tests.rs
+expression: "\n@deprecated(\"1\")\n@target(erlang)\n"
+---
+error: Syntax error
+  ┌─ /src/parse/error.gleam:2:1
+  │  
+2 │ ╭ @deprecated("1")
+3 │ │ @target(erlang)
+  │ ╰───────────────^ I was expecting a definition after this
```

Comment:
I think this error message will be confusing when there is a definition but it's one that is not supported, like external on a type.